python/tvm/tir/usmp/utils.py

```diff
@@ -83,14 +83,18 @@ class PoolAllocation(Object):
     pool_info : PoolInfo
         The PoolInfo to which this allocation corresponds to
 
+    byte_alignment : int
```

Comment:
We need more description here to explain why the byte offset is not sufficient, because naturally if the byte offset is calculated respecting the byte alignment, byte_alignment feels redundant.

Reply:
On having read the PR, I dont see where this is used. Can you point me if and where this is used ?